pkg/cluster/spec/tiflash.go

```diff
@@ -367,6 +368,32 @@ func (i *TiFlashInstance) initTiFlashConfig(cfg *scripts.TiFlashScript, clusterV
 		pathConfig = fmt.Sprintf(`path: "%s"`, cfg.DataDir)
 	}
 
+	if (semver.Compare(clusterVersion, "v4.0.12") >= 0 && semver.Compare(clusterVersion, "v5.0.0-rc") != 0) || clusterVersion == "nightly" {
```

Comment:
It's better to use `utils.Version(clusterVersion).IsNightly()` to check for nightly versions, the actual version in topo will be expanded from `nightly` to a specific version at that date.

Reply:
I push a commit to use `utils.Version(clusterVersion).IsNightly()` for TiFlash.

But I'm wondering should I also change this code in `playground.go`?
https://github.com/pingcap/tiup/blob/bd3cf97ef84f62be369b1c4af95b5901872c2803/components/playground/playground.go#L688

Reply:
I think so, it's better to change playground.go too.

Reply:
Addressed.